lib/cli.js

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env node
 'use strict';
 require('.')();
+
+// Override http networking to go through a proxy if one is configured.
+require('global-tunnel-ng').initialize();
```

Comment:
how to I access to global-tunnel-ng object during the runtime of a yeoman generator?